ring/partition_ring_status.gohtml

```diff
@@ -34,6 +42,19 @@
                         {{$ownerID}} <br />
                     {{ end }}
                 </td>
+                <td>
+                    <!-- Allow to force a state change -->
+                    {{ if and (not .Corrupted) (ne (index $stateChanges .State) 0) }}
+                        {{ $toState := index $stateChanges .State }}
+                        <form action="" method="POST" onsubmit="return confirm('Do you confirm you want to change the state to {{ $toState.CleanName }}?');">
+                            <input type="hidden" name="csrf_token" value="$__CSRF_TOKEN_PLACEHOLDER__">
```

Comment:
Note to reviewers: we do the same for other forms (e.g. ingester ring), but I can't find anywhere in dskit or Mimir where the token is injected and checked. There's a code comment mentioning it can be done through a middleware, so I've just added the token here as well, but I'm not sure who/where we use it.

Reply:
> it can be done

perhaps dskit authors thought that some external users will add this on their own? :)

Reply:
I don't know... these dskit authors, bad people :D